src/main/msp/msp.c

```diff
@@ -2646,6 +2654,16 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             sbufReadU8(src);
 #endif
         }
+        if (sbufBytesRemaining(src) >= 1) {
+#if defined(USE_GYRO_DATA_ANALYSE)
+            // Added in MSP API 1.44
+            gyroConfigMutable()->dyn_notch_count = sbufReadU8(src);
+            gyroConfigMutable()->dyn_notch_bandwidth_hz = sbufReadU16(src);
+#else
+            sbufReadU8(src);
+            sbufReadU16(src);
+#endif
+        }
```

Comment:
No need for the comment. This doesn't need it's own ```if()``` either. It can be put inside the existing ```if()``` for 1.44 here https://github.com/betaflight/betaflight/blob/a9802b92b5ce77a26ea2a8e2252b189a266fb71b/src/main/msp/msp.c#L2649 and increase the number of bytes to from 1 to 4 ```sbufBytesRemaining(src) >= 4```